visual-js/visual/src/utils.ts

```diff
@@ -122,3 +126,101 @@ export const parseRegionsForAPI = async <T>(
     ignoreElements: flattened.filter(isElementIn),
   };
 };
+
+/**
+ * Dom capturing script path
+ */
+
+export const domScriptPath = `${os.tmpdir()}/sauce-visual-dom-capture.js`;
+
+/**
+ * Fetch and save to tmp file dom capturing script
+ */
+
+export const downloadDomScript = async (api: ReturnType<typeof getApi>) => {
+  try {
+    const script = await api.domCaptureScript();
+    script && (await fs.writeFile(domScriptPath, script));
+  } catch (err: unknown) {
+    console.error(`Cannot get dom capturing script.\n${err}`);
+  }
+};
+
+/**
+ * Get dom capturing script from saved file
+ */
+
+export const getDomScript = async () => {
+  try {
+    return (await fs.readFile(domScriptPath)).toString();
+  } catch (err: unknown) {
+    if ((err as NodeJS.ErrnoException).code !== 'ENOENT') {
+      console.error(err);
+    }
+  }
+};
+
+export const removeDomScriptFile = async () => {
+  try {
+    await fs.unlink(domScriptPath);
```

Comment:
There will still be the folder created by `os.tmpdir()` present. We should probably remove it too.

Reply:
That folder is persistent / the value returned globally by node. I don't think we need to remove that folder since we're only creating a file in it and other processes / items could also append items here.